### ISSUE TITLE
Домашняя работа №1 - события и функции

### ISSUE THREAD

**Issue by redexp** [kravchuk-st/GeekHub-2020.js#1]
Тег `#formula` не найден

**Comment by kravchuk-st** [comment]
> ег `#formula` не найден

я ещё могу исправить, будет ли бот далее проверять мои домашки?

**Comment by redexp** [comment]
нет, не будет

**Comment by kravchuk-st** [comment]
> нет, не будет

с сертификатом я теперь тоже пролетаю?

**Comment by kravchuk-st** [comment]
> нет, не будет

с сертификатом я теперь тоже пролетаю? в требованиях не было условия про обязательность тега #formula, а остальное я выполнил

**Comment by redexp** [comment]
да, но делать для себя домашки и задавать вопросы можно. Ссылка на этот репозиторий в твоём резюме может оказаться более убедительной для работодателя, чем ссылка на сертификат. Отмазываться типа какие условия были не нужно, этим ты только делаешь себе хуже. 89 человек нормально поняли задание.

**Comment by kravchuk-st** [comment]
> да, но делать для себя домашки и задавать вопросы можно. Ссылка на этот репозиторий в твоём резюме может оказаться более убедительной для работодателя, чем ссылка на сертификат. Отмазываться типа какие условия были не нужно, этим ты только делаешь себе хуже. 89 человек нормально поняли задание.

я не отмазываюсь! зачем прописывать ещё один калькулятор, если можно всё посчитать одним, и менять бэкграунд по его результатам. но рад был поучаствовать, и удачи в  дальнейшем продвижении со студентами
